R/reportEDGETransport2.R

```diff
@@ -619,16 +749,16 @@ reportEDGETransport2 <- function(output_folder = ".", sub_folder = "EDGE-T/",
 
     #ES pkm are used as weights for data aggregation
     weight_pkm <- copy(demand_km)
-    setnames(weight_pkm, c("value","year"), c("weight","period"))
+    setnames(weight_pkm, c("value", "year"), c("weight", "period"))
 
     weight_pkm[, sector := ifelse(sector %in% c("Pass"), "trn_pass", "trn_freight")]
     weight_pkm[, sector := ifelse(subsector_L3 == c("International Aviation"), "trn_aviation_intl", sector)]
     weight_pkm[, sector := ifelse(subsector_L3 == c("International Ship"), "trn_shipping_intl", sector)]
 
     #Mapping for region Aggregation
     RegAggregation <- data.table(
-    aggr_reg = c("EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "NEU", "NEU"),
-    region = c("ENC", "EWN", "ECS", "ESC", "ECE", "FRA", "DEU", "UKI", "ESW", "NES", "NEN"))
+      aggr_reg = c("EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "EUR", "NEU", "NEU"),
```

Comment:
these lists should be loaded from the GDX similar to what is done in the main reporting:
https://github.com/pik-piam/remind2/blob/master/R/convGDX2MIF.R#L28